server/app_engine.py

```diff
@@ -1288,7 +1311,22 @@ def server_error(e):
 def exception_handler(e):
     add_userinfo_breadcrumb()
     capture_exception(e)
-    return flask.render_template('displayException.html', error_message=repr(e), version=application_properties['version']), 200
+    return flask.render_template('displayException.html',
```

Comment:
Can you change this so that it doesn't return html and 200 response code if the request method is 'POST'? The client side is expecting JSON if the response code is 200. I think for a POST request, you can just raise e.

Reply:
We don't want re-raise the exception because this is what's preventing a wall of stack trace text from being presented to the user.  We get the stack trace through sentry.  

I've changed this however to return 500 in all cases.

Reply:
Both submitTeam, and login routes are either GET or POST.  We can reevaluate these later if you like.